scst-store/amr/auth-k8s-sa-user-defined.hbs.md

```diff
@@ -53,9 +53,9 @@ metadata:
     kubernetes.io/service-account.name: amr-observer
 ```
 
-If you saved this to a file `observer-rbac.yaml` you can run `kubectl apply -f observer-rbac.yaml` to set everything up. If you prefer short lived service account tokens, remove the secret from the file beforehand and after creating the resources run `kubectl create token amr-observer-editor-token -n amr-observer-system` to create a token.
+If you saved this to a file `observer-rbac.yaml` you can run `kubectl apply -f observer-rbac.yaml` to set everything up. If you prefer short-lived service account tokens, remove the secret from the file beforehand and after creating the resources run `kubectl create token amr-observer-editor-token -n amr-observer-system` to create a token.
```

Comment:
`remove the secret from the file beforehand and after creating the resources run` -> `remove the secret from the file beforehand and after creating the resources, and then run`